test/engines_test.js

```diff
@@ -49,7 +49,7 @@ describe('Engines', function () {
         assert(asset.toString().match(/__output\.push/));
       });
     });
-
```

Comment:
this should be empty line, not a line with couple of spaces. It just helps to visually separate describe blocks.